influxql/engine.go

```diff
@@ -314,7 +314,7 @@ loop:
 	for _, row := range rows {
 		for _, values := range row.Values {
 			t := time.Unix(0, values[0].(int64))
-			values[0] = t.UTC().Format(time.RFC3339Nano)
+			values[0] = t.UTC()
```

Comment:
I was the one that made that change but now that I understand the behavior of the system better, clearly was the wrong spot.  We'll have to add custom marshaling to the influxql.Row struct to get time to marshal correctly.  Are we going to allow storing of time in actual data values as well?  If so, then we will have to have knowledge of all of that from the meta store when we marshal anyway.